src/components/member-profile/contribution-type/index.js

```diff
@@ -5,8 +5,14 @@ import classNames from '@components/member-profile/contribution-type/contributio
 import Contribution from '@components/member-profile/contribution/';
 import ActiveTask from '@components/member-profile/active-task';
 
-const renderContributions = (contributions, fullName, imageLink, devUser) =>
-  contributions.map((noteWorthyContribution, index) => (
+const ASSIGNED = 'ASSIGNED';
+
+const renderContributions = (contributions, fullName, imageLink, devUser) => {
+  const allContributions = contributions.filter(
```

Comment:
Is Contribution always an array?
Have we checked the edge case?

Reply:
Yeah checked for it, It always returns an array